src/Makefile

```
@@ -169,4 +169,16 @@ cadical-download:
 doc :
 	doxygen
 
-.PHONY: minisat2-download cudd-download glucose-download cadical-download
+install: all
+	for b in \
+		cbmc crangler \
+		goto-analyzer goto-cc goto-diff goto-instrument goto-harness \
+		symtab2gb ; do \
+		  cp $$b/$$b $(PREFIX)/bin/ ; \
+			cp ../doc/man/$$b.1 $(PREFIX)/doc/man/man1/ ; \
+	done
+	ln -sf goto-cc $(PREFIX)/bin/goto-gcc
+	ln -sf goto-cc $(PREFIX)/bin/goto-ld
```

Every other binary seems to be copied, but these two seem to be symlinks to the `goto-cc` produced as a compilation artifact. Is my understanding correct?

If it is, what happens in the case when the user deletes the build folder (or all of the project folder) for any reason. Won't these symlinks get broken?

---

Yeah, I think these `ln` command need to be run in the `$(PREFIX)/bin` directory, so that the _copied_ binaries are the ones linked to - either that or use this pattern:
```suggestion
	ln -sf $(PREFIX)/bin/goto-cc $(PREFIX)/bin/goto-ld
```

---

I have picked up this suggestion to avoid the doubt that Fotis rightly raised.